geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -84,7 +84,7 @@ <h3>
             <section class="transcription-link">
                 {% for ed in document.digital_editions %}
                     {% if ed.content.text %}
-                        <p><a href="{% url "corpus:document-transcription-text" document.pk ed.pk %}">Download {% for auth in ed.source.authorship_set.all %}{% include "snippets/comma.html" %}{{ auth.creator.last_name }}{% endfor %}'s edition.</a></p>
+                        <p><a href="{% url "corpus:document-transcription-text" document.pk ed.pk %}">Download {% if ed.source.authorship_set.count > 0 %}{% for auth in ed.source.authorship_set.all %}{% include "snippets/comma.html" %}{{ auth.creator.last_name }}{% endfor %}'s {% endif %}edition.</a></p>
```

Comment:
is `for` / `empty` useful at all here? admittedly "[unknown]'s transcription" doesn't look great. But shouldn't we indicate that the transcription is from an unknown source?

Reply:
I never knew about `{% empty %}` before, that's so useful! I see now it's used on the needs review template as well 😄 

Done in 3683dc1 👍